configs/data/euroc-default.yaml

```diff
@@ -0,0 +1 @@
+euroc-vicon-default.yaml
```

Comment:
Per-dataset yamls make sense, but why have `euroc-vicon-default` and `euroc-default`? The nesting seems to be too much. Why not just include the dataset directly in `data-default`?